src/admin/tests/test_watcher.py

```diff
@@ -0,0 +1,203 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for test_watcher.
+"""
+
+from collections import namedtuple
+import requests
+import unittest
+from unittest.mock import patch
+
+from lsst.qserv.admin.watcher import (
+    CheckFailure,
+    first_check_msg_t,
+    msg_prefix_t,
+    notify_data,
+    notify_headers,
+    query_msg_t,
+    show_query_msg_t,
+    start_msg_t,
+    stop_msg_t,
+    Watcher,
```

Comment:
Asking mostly out of curiosity. I'm seeing the extra comma at the last element in nearly all lists and tuples of this file. AFAIK it's not formally required by the language standard. In other languages, this would be a syntax error. Why it's here? Is there any specific requirement imposed by the LSST Style Guide, or this is just a "good practice" based on "how people usually do things"?

Reply:
[pep-8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas) has this to say about trailing commas:
> they are often helpful when a version control system is used, when a list of values, arguments or imported items is expected to be extended over time. The pattern is to put each value (etc.) on a line by itself, always adding a trailing comma, and add the close parenthesis/bracket/brace on the next line

I've been using a code formatting tool called black ("The Uncompromising Code Formatter") that Just Does all the annoying formatting that the pythonistas would otherwise snark at, and its rule is no trailing comma=list on one line, and trailing comma=each element on a line. If the list is too long for one line it'll even rewrite the list to be one element per line (with trailing comma). So, wookie/win, & I get to 🍺 that much sooner 😄